### DISCUSSION TITLE
Snap to end of month

### DISCUSSION THREAD

**Discussion by Toshinko** [visjs/vis-timeline#1570] [Q&A]
I want to snap events, I create and expand with the sideways arrows to the first day of month and the LAST day.
Eg: 2023-02-01 00:00:00 to 2023-02-28 23:59:59
Is this possible?
Thanks in advance.

**Comment by ProIntegritate** [comment]
You can add a month to the current end time, then set date to day one, then subtract one day i.e. 2023-03-01 - 1 =  2023-02-28.
